game/BeepSaber_Game.gd

```diff
@@ -651,4 +760,34 @@ func _on_Pause_Panel_continue_button():
 	continue_map()
 
 
+# FIXME this doesn't always seem to get called :(
```

Comment:
hmmm not sure why this was an issue... I could just save the highscore file each time we make an update. That's probably not that costly.

Reply:
yes; saving for each update should not be a problem